SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslationManager.cpp

```diff
@@ -101,6 +100,13 @@ void SpatialVirtualWorkerTranslationManager::ReclaimPartitionEntities()
 	QueryForServerWorkerEntities();
 }
 
+void SpatialVirtualWorkerTranslationManager::Advance(const TArray<Worker_Op>& Ops)
```

Comment:
This is ticked by the WellKnownEntity subview, how can we guarantee that the filter ops here always include everything we care about?

Reply:
I don't think there is any filtering applied to ops in subview deltas, is there? There is a comment at ViewDelta.cpp@31 which implies the opposite: `// No projection is applied to worker messages, as they are not entity specific.`